modules/widgets/simple-payments/form.php

```diff
@@ -10,23 +18,23 @@ class="widefat jetpack-simple-payments-widget-title"
 		value="<?php echo esc_attr( $instance['title'] ); ?>" />
 </p>
 <p class="jetpack-simple-payments-products-fieldset" <?php if ( empty( $product_posts ) ) { echo 'style="display:none;"'; } ?>>
-	<label for="<?php echo $this->get_field_id('product_post_id'); ?>">
+	<label for="<?php echo intval( $this->get_field_id( 'product_post_id' ) ); ?>">
```

Comment:
What do you think about using `esc_attr` instead? And integer as a post ID is quite a safe assumption, but it is an assumption anyway.

Reply:
Works for me. Fixed in a09ccd2614ef9ac8e413d65aeb900b584f505d7d